src/operator/nn/pool.h

```diff
@@ -687,7 +687,7 @@ inline void pool(mshadow::Stream<cpu>* s, const DType* in_data, const TShape& is
       LOG(FATAL) << "Unknown pooling type " << pool_type;
     }
   } else {
-    LOG(FATAL) << "Unsupported " << kernel.ndim() << "-D pooling";
+    LOG(FATAL) << "Unsupported " << kernel.ndim() << "-D non-avg pooling";
```

Comment:
Why non-avg here?

Reply:
I used to misunderstand the logic there and I change it back now. It might be fine now.